pkg/meta/base.go

```diff
@@ -566,15 +568,14 @@ func (m *baseMeta) startDeleteSliceTasks() {
 }
 
 func (m *baseMeta) stopDeleteSliceTasks() {
-	m.Lock()
+	m.dSliceMu.Lock()
```

Comment:
when dslices is full, this will be blocked, we should make sure that sessCtx is cancel before calling this.